Client/mods/deathmatch/logic/luadefs/CLuaPedDefs.cpp

```diff
@@ -149,6 +150,7 @@ void CLuaPedDefs::AddClass(lua_State* luaVM)
     lua_classfunction(luaVM, "getTask", "getPedTask");
     lua_classfunction(luaVM, "getTotalAmmo", "getPedTotalAmmo");
     lua_classfunction(luaVM, "getVoice", "getPedVoice");
+    lua_classfunction(luaVM, "resetVoice", "resetPedVoice");
```

Comment:
Please use the new argument parser, as described here: https://github.com/multitheftauto/mtasa-blue/wiki/Lua-API#how-to-add-a-new-lua-definition